src/language-css/printer-postcss.js

```diff
@@ -515,6 +515,27 @@ function genericPrint(path, options, print) {
           continue;
         }
 
+        // Ignore escape `\` in Less
+        if (
+          options.parser === "less" &&
+          iNode.value &&
+          iNode.value.indexOf("\\") !== -1
+        ) {
+          continue;
+        }
+
+        // Ignore escaped `/` in Less
+        if (
+          options.parser === "less" &&
```

Comment:
Also need add tests to pure `css` and `scss`, maybe no need check `options.parser= "less"`

Reply:
Yes, it worked without `options.parser === "less"`

Reply:
@sh7dm let's add test to `css` and `scss`, also please provide example in comment as in other places (example - `Ignore escape "\" in Less mixins (example "label { .margin-bottom-1\\/3; }")`)

Reply:
@evilebottnawi where to put css test?

Reply:
@sh7dm `scss` https://github.com/prettier/prettier/tree/master/tests/css_scss and `css` https://github.com/prettier/prettier/blob/master/tests/css_character_escaping/character_escaping.css

Reply:
b75ecca